src/exports.jl

```diff
@@ -67,6 +67,8 @@ export
     ExclusiveTopology,
     getneighborhood,
     faceskeleton,
+    compute_vertex_star_stencils,
```

Comment:
maybe just `vertex_star_stencil` ? `compute` is kinda true in any case with computers

Reply:
Not really. For example, getters do not compute stuff.

Reply:
I am also not really understanding the point here, we have several functions even in Ferrite with this prefix https://github.com/search?q=repo%3AFerrite-FEM%2FFerrite.jl%20compute_&type=code

Reply:
Those are mostly not Ferrite exposed functions. Its a better name without the compute imo.

Reply:
To clarify: we would never write compute_function_value or compute_function_divergence since it would be a prefix which is to some extent almost true whenever you call something that is not 100% cached. So where do you draw the line and what is the advantage over documenting in the docs that it's actually computed on the fly